research-software-directory/RSD-as-a-service
# Include Roboto fonts to default index.css

# Include Roboto and Work Sans fonts into default index.css and the font woff2 files

Changes proposed in this pull request:

* The fonts defined in default styles of RSD include Roboto font. For the users who do not have Roboto fonts locally the organisation card is not shown with proper font and font-weight.      
* The font files are placed in the `frontend/public/styles` folder. This folder is part of docker-compose volume mounts and can be overwritten by custom settings.

How to test:

* `make start` to rebuild everything
* confirm Roboto font is included by navigating to organisation overview, the score fonts should look like in the screenshot

## Example Roboto font applied

![image](https://user-images.githubusercontent.com/9204081/205513893-14c452b7-d4bc-4278-9227-69b39ebdaa5a.png)

![image](https://user-images.githubusercontent.com/9204081/205513977-a8ebde93-ed10-47eb-8b89-1f504e56db6c.png)


PR Checklist:
